sdk/ownership/backend.go

```diff
@@ -16,18 +16,21 @@ const backendName = "ownership"
 
 // Backend is the ownership backend.
 type Backend struct {
-	storeKey *cosmostypes.KVStoreKey
+	storeKey               *cosmostypes.KVStoreKey
+	storeIndexRessourceKey *cosmostypes.KVStoreKey
 }
 
 // NewBackend returns the backend of the ownership sdk.
 func NewBackend(appFactory *cosmos.AppFactory) *Backend {
 	backend := &Backend{
-		storeKey: cosmostypes.NewKVStoreKey(backendName),
+		storeKey:               cosmostypes.NewKVStoreKey(backendName),
+		storeIndexRessourceKey: cosmostypes.NewKVStoreKey(backendName + ".index.resourceHash"),
```

Comment:
After some debate with @antho1404 we decide to not implement any index for now.
Let's give us the opportunity to improve the performance and efficiency later.

Also, the relationship between resources and owners is many to many (even if for now it's one to one because of logic limitation). This index you implemented is only one to one.